frontend/src/components/Button/index.module.css

```diff
@@ -1,12 +1,16 @@
 .button {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  padding: 15px 40px;
-  margin-top: 64px;
-  font-family: "Libre Baskerville", serif;
-  font-size: 20px;
-  background: #d9d9d9;
-  border: 1px solid #000;
-  border-radius: 5px;
-}
+  width: 335px;
+  height: 60px;
```

Comment:
We don't want fixed with/height buttons. The button width should be fluid, meaning it's size is a result of the space needed to render whatever text content we have in there plus the padding around it